src/overmap.h

```diff
@@ -473,12 +486,17 @@ class overmap
 extern std::unordered_map<std::string,oter_t> otermap;
 extern std::vector<oter_t> oterlist;
 //extern const regional_settings default_region_settings;
-extern std::unordered_map<std::string, regional_settings> region_settings_map;
+typedef std::unordered_map<std::string, regional_settings> t_regional_settings_map;
+typedef t_regional_settings_map::iterator t_regional_settings_map_itr;
```

Comment:
Or just use `auto` where you need an iterator: `auto iter = theMap.find( ... )` or `for( auto iter = theMap.begin()`